.github/workflows/run-tests.yml

```
@@ -0,0 +1,62 @@
+name: Run Tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - python-version: 2.7
+            tox-env: py27
+          - python-version: 3.6
+            tox-env: py36
+          - python-version: 3.7
+            tox-env: py37,docs,readme,black
+          - python-version: pypy3
+            tox-env: pypy3
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install test dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools
+        python -m pip install tox
+    - name: Install coveralls dependencies
+      if: ${{ matrix.python-version != '2.7' }}
+      run: |
+        python -m pip install coveralls coverage-lcov toml
+    - name: Execute tests with tox
+      env:
+        TOXENV: ${{ matrix.tox-env }}
+      run: |
+        tox
+    - name: Coverage format into lcov
+      if: ${{ matrix.python-version != '2.7' }}
+      run: |
+        coverage-lcov --output_file_path lcov.info
+    - name: Coveralls Parallel
+      uses: coverallsapp/github-action@master
```

Not sure if this matters, but in my own repos I've had less problems when setting this value with an actual version number, I think 1.1.3 is latest stable:

```suggestion
      uses: coverallsapp/github-action@1.1.3
```